extensions/bindings.md

```diff
@@ -5,7 +5,7 @@ Bindings are exposed inside of a container during all three phases of the lifecy
 During the detect and build phases, the lifecycle MUST provide any bindings as files in `<platform>/bindings/<binding-name>` with directory names matching the name of the binding.  Binding names MUST match `[a-z0-9\-\.]{1,253}`.
 
 ### Metadata
-Within each binding directory the lifecycle MUST provide a `metadata` directory containing `kind`, `provider`, and `tags` files.  The value of the `kind` file MUST contain an abstract classification of the binding.  The value of the `provider` file MUST identify the provider of this binding.  The value of the `tags` file MUST contain a newline delimited collection of tags attached to the binding.
+Within each binding directory the lifecycle MUST provide a `metadata` directory containing `kind` and `provider` files.  The value of the `kind` file MUST contain an abstract classification of the binding.  The value of the `provider` file MUST identify the provider of this binding.
```

Comment:
Can we make provider optional? There are cases where it is useful but I don't think it should be required.

Reply:
Should that just be a separate PR?

Reply:
a separate PR is fine with me

Reply:
I don't think that it should be optional.  I think the file should be required, but doesn't necessarily need to have a value within it.

Reply:
Was there a resolution to this?

Reply:
No... @nebhale and I argued some more on a call but continue to disagree ;). I think we can separate that discussion from this PR.